pkg/test/framework/components/istio/operator.go

```diff
@@ -518,37 +537,27 @@ func installRemoteClusters(i *operatorComponent, cfg Config, cluster resource.Cl
 			return err
 		}
 		installSettings = append(installSettings,
-			"--set", "values.global.remotePilotAddress="+remoteIstiodAddress.IP.String())
+			"--set", "values.global.remotePilotAddress="+remoteIstiodAddress.IP.String(),
+			"--set", fmt.Sprintf("values.istiodRemote.injectionURL=https://%s:%d/inject/net/%s/cluster/%s",
```

Comment:
In a real-world install would we have to specify these in `set` flags? Can you leave a comment that we either plan to clean this up, or are doing this as a workaround for our test env.

Reply:
yes, this is needed even in real-world in current code base.  So when user install remote cluster, they need set this via IOP or --set after they get the eastwest GW address in primary cluster.  

So  this is not a workaround.  If we plan to clean this up, we may need a seperate design doc to discuss how to land this.

Reply:
one proposal is to generate the url based on discoverAddr and cluster and network info, which we have all that info from other places.  I agree we should look into simplify this post 1.8.  I'll open a WI to track it.